applications/machine_learning/app_desc.rst

```diff
@@ -0,0 +1,524 @@
+
+By default, the following buttons are used by the application:
+
+* Thingy:53:
+
+  * The **SW3** button switches between data forwarding and running the machine learning model.
+
+* nRF52840 and nRF5340 DKs:
+
+  * **Button 1** switches between data forwarding and running a machine learning model.
+  * **Button 3** changes the signal generated by the simulated sensor.
+
+LEDs
+====
+
+The application uses one LED to display the application state.
+The LED displays either the state of data forwarding or the machine learning prediction results.
+You can configure the LED effect in the application configuration files.
+
+If the application uses the simulated sensor signal, it uses another LED to display the effect that represents the signal generated by the simulated sensor.
+The application defines common LED effects for both the machine learning results and the simulated sensor signal.
+
+By default, the application uses the following LED effects:
+
+* Thingy:53 displays the application state in the RGB scale using **LED1**.
+
+  * If the device is returning the machine learning prediction results, the LED uses the following predefined colors:
+
+    * ``rotate`` - Red
+    * ``updown`` - Green
+    * ``tap`` - Blue
+    * Anomaly - Purple
+
+    If the machine learning model is running but has not detected anything yet or the ``idle`` state is detected, the LED is blinking.
+    After a successful detection, the LED is set to the predefined color.
+    The LED effect is overridden on the next successful detection.
+  * If the device forwards data, the LED color turns red and uses the following blinking patterns:
+
+    * LED blinks slowly if it is not connected.
+    * LED blinks with an average frequency if it is connected but is not actively forwarding data.
+    * LED blinks rapidly if it is connected and is actively forwarding data.
+
+* Both nRF5340 DK and nRF52840 DK use monochromatic LEDs to display the application state.
+  The **LED1** displays the application state, and the **LED2** displays the signal generated by the simulated sensor.
+
+  * If the device is returning the machine learning prediction results, the **LED1** blinks for a predefined number of times and then turns off for a period of time.
+    Then the sequence is repeated.
+    The machine learning result is represented by the number of blinks:
+
+    * ``sine`` - 1 blink
+    * ``triangle`` - 2 blinks
+    * ``square`` - 3 blinks
+    * ``idle`` - 4 blinks
+
+    If the machine learning model is running but has not detected anything yet or has detected an anomaly, the **LED1** is breathing.
+  * If the device forwards data, the **LED1** uses the following blinking patterns:
+
+    * LED blinks slowly if it is not connected.
+    * LED blinks with an average frequency if it is connected but is not actively forwarding data.
+    * LED blinks rapidly if it is connected and is actively forwarding data.
+
+.. _nrf_machine_learning_app_configuration:
+
+Configuration
```

Comment:
Just a suggestion, I don't know what templates say about it, but e.g. nRF Dekstop has a separate page covering configuration: https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/applications/nrf_desktop/application_kconfig.html#nrf-desktop-application-specific-kconfig-options
In fact Machine Learning has a few Kconfig files in src/ src/modules and src/utils so maybe there is enough content.

Reply:
Good point. I will create a new page for the configuration options. We follow the same for nRF5340 Audio applications as well - https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/applications/nrf5340_audio/index.html.